service/src/main/java/bio/terra/workspace/app/controller/ResourceController.java

```diff
@@ -95,6 +99,14 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
     return new ResponseEntity<>(apiResourceList, HttpStatus.OK);
   }
 
+  @Override
+  public ResponseEntity<Boolean> validateReference(UUID workspaceId, UUID resourceId) {
+    AuthenticatedUserRequest userRequest = getAuthenticatedInfo();
+    boolean isValid =
+        referencedResourceService.validateReference(workspaceId, resourceId, userRequest);
+    return new ResponseEntity<>(isValid, HttpStatus.OK);
```

Comment:
This is a JSON object - it's just a single boolean field instead of a complex object. I could wrap it in another object, but I'm not sure that really adds anything.